src/components/AssetCard/AssetCard.css

```diff
@@ -0,0 +1,368 @@
+
+.AssetCard {
+  height: 345px;
+  overflow: visible;
+}
+
+.AssetCard.ui.card.AssetCard.catalog {
+  height: 360px;
+}
+
+.AssetCard.ui.card > .content.catalog {
+  padding: 14px 14px 18px 14px;
+}
+
+.AssetCard .header .catalogTitle {
+  flex: 1 2 auto;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  margin-bottom: 0px;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  line-height: 24px;
+}
+
+.AssetCard .textOverflow {
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.AssetCard.ui.card.AssetCard > .content,
+.AssetCard.ui.cards > .card.AssetCard > .content {
+  height: 185px;
+}
+
+.AssetCard.ui.card.AssetCard > .content > .header:not(.ui),
+.AssetCard.ui.cards > .card.AssetCard > .content > .header:not(.ui) {
+  flex: unset;
+}
+
+.AssetCard .header,
+.AssetCard .dcl.mana.inline {
+  font-size: 17px;
+  line-height: 26px;
+  letter-spacing: -0.2px;
+}
+
+.AssetCard .ui.header.small {
+  font-size: 14.5px;
+  font-weight: 400;
+  margin-top: -1px;
+}
+
+.AssetCard .CatalogItemInformation {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  transition: inherit;
+  flex: 1;
+}
+
+.AssetCard .CatalogItemInformation > span > img {
+  margin-left: 4px;
+}
+
+.AssetCard .header .title {
+  flex: 1 2 auto;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  margin-right: 20px;
+  margin-bottom: 0px;
+  display: flex;
+  flex-direction: column;
+}
+
+.AssetCard .creator {
+  font-size: 14px;
+  font-weight: 400;
+  color: var(--secondary-text);
+}
+
+.AssetCard.ui.card.AssetCard > .content > .header:not(.ui),
+.AssetCard.ui.cards > .card.AssetCard > .content > .header:not(.ui) {
+  display: flex;
+  margin-bottom: 0px;
+}
+
+.AssetCard .dcl.mana.inline {
+  margin: 0px;
+  flex: none;
+}
+
+.AssetCard.ui.cards > .ui.card.AssetCard.link .meta,
+.AssetCard.ui.card.link .meta {
+  margin-bottom: 4px;
+}
+
+.AssetCard .tags {
+  margin-top: 10px;
+}
+
+.AssetCard .sub-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: var(--text);
+}
+
+.AssetCard .card-meta {
+  margin-right: 10px;
+}
+.AssetCard .NotForSale {
+  font-weight: bold;
+  margin-top: 12px;
+  margin-bottom: 4px;
+}
+
+.AssetCard .PriceInMana .ui.header.large {
+  font-size: 30px;
+  font-weight: 600;
+  margin: 0px;
+  display: flex;
+  flex-direction: row;
+}
+
+.AssetCard .ens-subdomain {
+  overflow: hidden;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.AssetCard .dcl.atlas {
+  overflow: hidden;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.ui.card.AssetCard .AssetImage,
+.ui.card.AssetCard .AssetImage .rarity-background {
+  border-radius: 10px 10px 0px 0px !important;
+}
+
+.AssetCard .AssetImage .ens-subdomain {
+  background-size: 557px;
+  background-position-y: -133px;
+}
+
+.AssetCard .AssetImage .ens-subdomain .monospace {
+  margin-top: 0px;
+}
```

Comment:
These two might no be needed as we don't have names, right? Would you mind removing them?